Update aero_installer.sh

osluys opened the pull request:
- More robust error management, especially when reinstalling after failure
- Better LSB support for LibreOffice headless and AerooDOCS _init.d_ services
- Uninstall option
- General script improvements

osluys committed:
Update aero_installer.sh

- General script improvements
- Uninstall option
- More robust error management, especially during reinstall after failure
- Better LSB support for LibreOffice headless and AerooDOCS init.d services

osluys committed:
Update README.md

osluys committed:
Update README.md

osluys committed:
Separate out `addons` dir from main Odoo dir

osluys committed:
Update README.md

osluys committed:
Update README.md

osluys committed:
Use `unlink` for removing symlinks

osluys committed:
Merge branch 'Stable' of https://github.com/osluys/AerooInstallScript into Stable

osluys committed:
Fix argument passing to LibreOffice executable

Yenthe666 commented:
@osluys nice, thanks a lot! I will give this a try as soon as I can. If everything works out fine I'll merge this :+1:

osluys committed:
Replace script name reference with constant value

Determining the name of the init.d script from the `$0` variable could
cause potential problems when the script is called indirectly via another
command, so avoid this problem by specifying a constant value instead.

osluys commented:
I should mention that I've only tested it on Ubuntu 15.10. I was attempting to use your script to install Aeroo for Odoo v9 and encountered a few problems, so I ended up making a few changes but maybe I got a little carried away. Thanks, by the way, for your initial script and your mods to get Aeroo working with v9: after I got the install script working, I was able to generate reports with Aeroo without any difficulties.

osluys committed:
Add libreoffice- calc, core, writer dependencies
